poll/public/css/poll.css

```diff
@@ -373,3 +373,39 @@ th.survey-answer {
         line-height: 24px;
     }
 }
+
+ /*RTL specified CSS for polls & survey*/
+
+[dir="rtl"] .poll-input-container, [dir="rtl"] .poll-answer-text{
```

Comment:
@murad-hubib Can we use the `rtl` class instead of `[dir="rtl"]`, like in the [chat xblock](https://github.com/open-craft/xblock-chat/blob/fc82c9d23946b2f657034fbaab0adea7a1233479/chat/public/css/chat.css#L386)?
The drag and drop v2 which is included in edx-platform by default is [also using the `rtl` class](https://github.com/edx-solutions/xblock-drag-and-drop-v2/blob/8c6ba95b5eccb7a6c9a7e860ddd1e19cef82ab10/drag_and_drop_v2/public/css/drag_and_drop.css#L746).

Reply:
Yes we can add class instead of dir property. I am going to commit the change in a while.

Reply:
@mtyaka I have commit the change (rtl class intead of [dir="rtl"]) right in this PR but seems like you  have already merged it. May you please take care of the change or should I open an other PR?

Reply:
@murad-hubib This PR hasn't been merged yet. I verified it and will merge it now & cut a new release.